### PR TITLE
add support for pubsub subscription

### PR DESCRIPTION
**Pull request by ericyz**

### CODE REVIEW

**Comment by ericyz** [comment]
Hi @melinath , this PR is ready for review

File: converters/google/mappers.go [hunk 1]
```diff
@@ -50,17 +50,20 @@ type mapper struct {
 func mappers() map[string][]mapper {
 	return map[string][]mapper{
 		// TODO: Use a generated mapping once it lands in the conversion library.
-		"google_compute_firewall":      {{convert: converter.GetComputeFirewallCaiObject}},
-		"google_compute_disk":          {{convert: converter.GetComputeDiskCaiObject}},
+		"google_compute_firewall":               {{convert: converter.GetComputeFirewallCaiObject}},
+		"google_compute_disk":                   {{convert: converter.GetComputeDiskCaiObject}},
 		"google_compute_instance":      {{convert: converter.GetComputeInstanceCaiObject}},
 		"google_storage_bucket":        {{convert: converter.GetStorageBucketCaiObject}},
```
Comment (melinath) [inline converters/google/mappers.go:56]: It looks like these two keys need to be deleted - currently causing test compilation to fail.

**Review by melinath — CHANGES_REQUESTED** [review]

**Review by melinath — CHANGES_REQUESTED** [review]
Test failures:

```
FAIL: TestCLI/v=0.12/tf=example_pubsub_subscription/offline=false/cmd=convert (0.00s)
            cli_test.go:244: unmarshaling: json: cannot unmarshal object into Go value of type []google.Asset
FAIL: TestCLI/v=0.12/tf=example_pubsub_subscription/offline=true/cmd=convert (0.00s)
            cli_test.go:244: unmarshaling: json: cannot unmarshal object into Go value of type []google.Asset
```

I think the `.tfplan.json` and `.json` files might be flipped.

**Comment by ericyz** [comment]
> Test failures:
> 
> ```
> FAIL: TestCLI/v=0.12/tf=example_pubsub_subscription/offline=false/cmd=convert (0.00s)
>             cli_test.go:244: unmarshaling: json: cannot unmarshal object into Go value of type []google.Asset
> FAIL: TestCLI/v=0.12/tf=example_pubsub_subscription/offline=true/cmd=convert (0.00s)
>             cli_test.go:244: unmarshaling: json: cannot unmarshal object into Go value of type []google.Asset
> ```
> 
> I think the `.tfplan.json` and `.json` files might be flipped in terms of filename vs content.

Thanks! Fixed the those files

**Review by melinath — APPROVED** [review]
lgtm

**Pull request merged**